### DISCUSSION TITLE
.NET 7 Ideas For WinForms

### DISCUSSION THREAD

**Discussion by TrevorMcFadden** [dotnet/winforms#6850] [Ideas] [answered]
Hello all! I think for .NET 7 and especially for Windows Forms' .NET Framework projects, one of the biggest changes should be having WebView2 be the default WebBrowser function. Currently, the default WebBrowser function is based upon IE's Trident engine, which isn't how most websites operate anymore. But, by having a Chromium-based engine incorporated into the Toolbox by default (as opposed to adding it via Nuget), a developer could easily add a modern web browsing experience to their project. Another idea I think would be helpful in future WinForms projects is adding more button customization, specifically corner radius and icon development. Working with UWP, I find that WinForms could use something similar in developing better UI. It would be nice if WinForms had the capacity to match Windows 11's 'rounded' design (versus matching Windows 10's 'square' design). Other than that, keep up the good work!

Grace and peace,
Trevor McFadden

**Comment by KlausLoeffelmann** [comment, marked as answer]
Please take a look at my answer for this issue:
https://github.com/dotnet/winforms/discussions/6709
